src/tests/ftest/control/dmg_telemetry_basic.yaml

```diff
@@ -2,7 +2,7 @@ hosts:
   test_servers: 2
   test_clients: 1
 timeouts:
-  test_telemetry_list: 60
+  test_telemetry_list: 90
```

Comment:
The test was killed after 120 sec (90sec test timeout + 30sec that is added for agent/server start/stop to every test).  It looks like the test was still running at 120 sec.    Try a debug run were you increase the timeout to 600 sec and then determine the amount of time the test actually takes to complete and see if this completion time is reasonable.

If the test used to pass in 60 secs and now it is taking 300 sec then maybe there is another issue that should be looked into.

Reply:
@mjean308 See my comment here
https://daosio.atlassian.net/browse/DAOS-15337

The problem is related to general VM slowness - not this test in particular